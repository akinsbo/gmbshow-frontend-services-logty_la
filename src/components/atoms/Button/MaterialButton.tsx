import Button from '@material-ui/core/Button';
import * as React from 'react'

// Uncomment if you wish to use interface for typing
interface LabelProps {
    children?: JSX.Element[] | JSX.Element | string | string[]
}

const MaterialButton = (props: LabelProps) => {
    return <Button>{props.children}</Button>
}

export default MaterialButton