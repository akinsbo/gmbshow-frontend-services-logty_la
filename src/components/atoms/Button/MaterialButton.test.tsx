
import * as Enzyme from "enzyme"
import enzymeAdapterReact16 from "enzyme-adapter-react-16"
import "jest-enzyme"
import * as React from "react"
import * as renderer from "react-test-renderer"
import MaterialButton from "./MaterialButton"

Enzyme.configure({
    adapter: new enzymeAdapterReact16()
})

describe("MaterialButton", () => {
    it("renders correctly", () => {
      const tree = renderer.create(
        <MaterialButton><h1>Word</h1></MaterialButton>
      ).toJSON()
      expect(tree).toMatchSnapshot()
    })
    
})