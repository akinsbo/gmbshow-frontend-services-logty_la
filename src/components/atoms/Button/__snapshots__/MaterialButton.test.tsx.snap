// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MaterialButton renders correctly 1`] = `
<button
  className="MuiButtonBase-root-27 MuiButton-root-1 MuiButton-text-3 MuiButton-flat-6"
  disabled={false}
  onBlur={[Function]}
  onContextMenu={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex="0"
  type="button"
>
  <span
    className="MuiButton-label-2"
  >
    <h1>
      Word
    </h1>
  </span>
  <span
    className="MuiTouchRipple-root-30"
  />
</button>
`;
